lib/game.rb

```diff
@@ -0,0 +1,21 @@
+class Game
```

Comment:
All of this works, and you could clean up the national_drawing functionality like this:

```ruby
class Game
  ...
  def initialize(name, cost, national_drawing = false)
    ...
    @national_drawing = national_drawing
  end
  
  def national_drawing?
    @national_drawing
  end
end
```